components/datetime/tests/fixtures/data/icu4x/dates/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
There shouldn't be a manifest.json in this directory

Reply:
If I removed it, `FsDataProvider` errors out with:

```
Loading file from testdata directory: IoError(Os { code: 2, kind: NotFound, message: "No such file or directory" })'
```

Reply:
You have two manifest.json files.  You need the one in the "icu4x" directory but not the one in the "dates" directory.